index.html

```diff
@@ -129,11 +129,11 @@
         window.addEventListener(
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
-    <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
+    <script src ="/config.js"></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
```

Comment:
please undo these, they are set like that intentionally.

Reply:
These changes were only made in order to test the feature flag and I was planning on removing them when everything else was checked of. I tried removing the ones other than interace_config but am getting missing file errors. I will work on trying to see if I can remove the unnecessary changes while keeping the local interface_config